### DISCUSSION TITLE
Duplicate a collection.

### DISCUSSION THREAD

**Discussion by HemaVGDS** [directus/directus#17424] [Feature Requests]
I have a page collection with many sections . Requesting a feature to clone a page with all available sections and inter-related fields .

**Comment by paescuj** [comment]
This has already come up a few times, e.g. in #10589.

**Reply by HemaVGDS** [reply]
Is there any way we can achieve this cloning? Any suggestion

**Reply by paescuj** [reply]
You could try to create a script using the [API](https://docs.directus.io/reference/system/collections.html) to clone / copy a collection with all fields.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
